docs/DeveloperGuide.adoc

```diff
@@ -104,7 +104,7 @@ image::Architecture.png[width="600"]
 The *_Architecture Diagram_* given above explains the high-level design of the App. Given below is a quick overview of each component.
```

Comment:
Add {nbsp} + after an image to get a blank line

Reply:
fixed